x/feegrant/fees.go

```diff
@@ -28,4 +28,7 @@ type FeeAllowanceI interface {
 
 	// ExpiresAt returns the expiry time of the allowance.
 	ExpiresAt() (*time.Time, error)
+
+	// ClearPeriodReset clear "PeriodReset" value by valid time
+	ClearPeriodReset(validTime time.Time) error
```

Comment:
Actually, I believe calling it `ClearPeriodReset` is a misnomer since we're updating the value (given by the argument). Instead, this should be something like `UpdatePeriodReset`

Reply:
When I initially started, I used 'ResetPeriodReset,' which seems odd in its naming. I agree with changing it to 'UpdatePeriodReset.' Either way, the purpose is to modify the value of the 'periodReset' field.

Reply:
yeah let's go with `UpdatePeriodReset`

Reply:
@alexanderbez I've already changed my name, please review it again. Also, there's an issue with the automated process below, what could be the reason for the errors?

Reply:
I've allowed to CI run. Let's hope it's green!